SystemServices/SystemServices.cpp

```diff
@@ -735,7 +736,8 @@ namespace WPEFramework {
                enableFwAutoreboot = (parameters["enable"].Boolean());
                LOGINFO("setFirmwareAutoReboot : %s\n",(enableFwAutoreboot)? "true":"false");
 
-               const char *set_rfc_val = (parameters["enable"].String().c_str());
+               std::string delaySeconds = parameters["enable"].String();
```

Comment:
Change the name of the variable "delaySeconds" since it is a different parameter.